plugin.video.vrt.nu/resources/language/resource.language.en_gb/strings.po

```diff
@@ -64,12 +64,8 @@ msgid "Woops something went wrong, check the log for more details"
 msgstr "Woops something went wrong, check the log for more details"
 
 msgctxt "#32061"
-msgid "SOCKS proxies are currently unsupported"
-msgstr "SOCKS proxies are currently unsupported"
-
-msgctxt "#32062"
-msgid "Using a SOCKS proxy requires the requests[socks] library installed."
-msgstr "Using a SOCKS proxy requires the requests[socks] library installed."
+msgid "Using a SOCKS proxy requires the PySocks library (script.module.pysocks) installed."
```

Comment:
can't we add the PySocks library in the addon.xml as a dependency?

Reply:
It is not available yet, and it may not be for some of our users in the future. Otherwise I would agree with you.